docs/time.md

```diff
@@ -249,21 +249,21 @@ time.minus({ minutes: 5, nanoseconds: 800 })  // => 19:34:09.068345405
 **Returns:** a `Temporal.Duration` representing the difference between `time` and `other`.
 
 This method computes the difference between the two times represented by `time` and `other`, and returns it as a `Temporal.Duration` object.
-A `RangeError` will be thrown if `other` is later than `time`, because `Temporal.Duration` objects cannot represent negative durations.
+If `other` is later than `time` then the resulting duration will be negative.
 
 The `largestUnit` parameter controls how the resulting duration is expressed.
 The returned `Temporal.Duration` object will not have any nonzero fields that are larger than the unit in `largestUnit`.
 A difference of two hours will become 7200 seconds when `largestUnit` is `'seconds'`, for example.
 However, a difference of 30 seconds will still be 30 seconds even if `largestUnit` is `'hours'`.
 
 The default largest unit in the result is technically `'days'`, for consistency with other Temporal types' `difference` methods.
-However, since time differences are always shorter than one day, largest units of `'years'`, `'months'`, or `'days'` are treated as `'hours'`.
+However, since this method never returns any duration longer than 12 hours, largest units of `'years'`, `'months'`, `'weeks'`, or `'days'`, are by definition treated as `'hours'`.
```

Comment:
The "12 hour" statement is not correct. The max is 24 hours. 
```
Temporal.Time.from('23:59').minus(Temporal.Time.from('00:00'));
```

Reply:
Ah, this appeared in there because I started from a revert of the commit in #667 and I didn't remove it.